### PR TITLE
replaces nutritionDisplay with nutritionStats and changes NutritionAPI

### LINKED ISSUE #69: replace nutritionDisplay with nutritionStats + update functionality
removes nutritionDisplay and creates nutritionStats use case which computes the stats for the home view

### PR DESCRIPTION
**Pull request by parkhangorodsky**
this PR:
- removes the nutrition print statements used in phase 1
- creates NutritionStats and deletes NutritionDisplay
- modifies NutritionJSONHandler and NutritionDisplayApi to use NutritionStats types and iterate over JSONObjects
- uses placeholder for shoppingList.getListRecipes (which will be used to compute the nutrition stats)

### CODE REVIEW

File: src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java [hunk 1]
```diff
@@ -36,38 +32,23 @@ default List<Nutrition> convertJSONtoNutritionList(JSONObject nutritionJSONObjec
...
-             nutrition_categories.add(sodium);
-             nutrition_categories.add(fats);
-
-             for (String category : nutrition_categories) {
-                 JSONObject totalNutrients = nutritionJSONObject.getJSONObject("totalNutrients");
 
-                 String label = totalNutrients.getJSONObject(category).getString("label");
-                 float quantity = totalNutrients.getJSONObject(category).getFloat("quantity");
-                 String unit = totalNutrients.getJSONObject(category).getString("unit");
+             JSONObject totalNutrients = nutritionJSONObject.getJSONObject("totalNutrients");
```
Comment (seonghyunban) [inline src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java:34]: Iterator? Consider for loop instead

File: src/main/java/use_cases/nutrition_stats/interface_adapeter/view_model/NutritionStatsViewModel.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        return nutritionalStatistics;
+    }
+
+    /**
+     * Notifies all registered listeners that the search results have changed.
+     * Fires a property change event with "search recipe" if there are results, otherwise fires "empty result".
+     */
+    public void firePropertyChanged() {
+        if (!nutritionalStatistics.getNutritionInfoAverage().isEmpty()) {
+            support.firePropertyChange("home view", null, this.nutritionalStatistics);
```
Comment (seonghyunban) [inline src/main/java/use_cases/nutrition_stats/interface_adapeter/view_model/NutritionStatsViewModel.java]: Name is updated "home view" -> "Home"

**Review by seonghyunban — APPROVED** [review]
Left comment for parts that need attention

File: src/main/java/use_cases/nutrition_stats/interface_adapeter/view_model/NutritionStatsViewModel.java (hunk 2 above)
Comment (parkhangorodsky) [reply src/main/java/use_cases/nutrition_stats/interface_adapeter/view_model/NutritionStatsViewModel.java]: done

**Pull request merged**